xml/System.Text.Encodings.Web/TextEncoder.xml

```diff
@@ -221,14 +221,22 @@
         <Parameter Name="isFinalBlock" Type="System.Boolean" Index="4" FrameworkAlternate="netcore-3.0" />
       </Parameters>
       <Docs>
-        <param name="source">To be added.</param>
-        <param name="destination">To be added.</param>
-        <param name="charsConsumed">To be added.</param>
-        <param name="charsWritten">To be added.</param>
-        <param name="isFinalBlock">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="source">A source buffer containing the characters to encode.</param>
+        <param name="destination">The destination buffer to which the encoded form of <paramref name="source" /> will be written.</param>
+        <param name="charsConsumed">The number of characters consumed from the <paramref name="source" /> buffer.</param>
+        <param name="charsWritten">The number of characters written to the <paramref name="destination" /> buffer.</param>
+        <param name="isFinalBlock"><see langword="true" /> if there is further source data that needs to be encoded; <see langword="false" /> if there is no further source data that needs to be encoded.</param>
```

Comment:
This needs to be inverted. cc @steveharter

Reply:
What do you mean by inverting? Do you mean true <-> false, like this?:

```suggestion
        <param name="isFinalBlock"><see langword="false" /> if there is further source data that needs to be encoded; <see langword="true" /> if there is no further source data that needs to be encoded.</param>
```

Reply:
Clarified in person. I'll commit this and fix the one for EncodeUtf8 separately.